README.md

```diff
@@ -1 +1,64 @@
-# Awesome-ES6
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+> Awesome Books is an application which you can add, delete and display your awesome books!
+
+## Built With
+
+- HTML5
+- CSS3
+- JavaScript
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+1- Copy this link git@github.com:ichala/Awesome-ES6.git <br>
+2- Get the directory that you want to clone the repository. <br>
+3- Open the command prompt in this directory. <br>
+4- Write git@github.com:ichala/Awesome-ES6.git <br>
+5- Go to the repository folder in your command prompt cd Awesome-ES6 <br>
+6- Run npm install to install the dev dependencies for linters and node sass.
+
+### Prerequisites
+
+Web Browser (Chrome,Safari,Mozilla,Edge)
+
+### Install
+
+No need to install. Open the index.html file with a browser.
+
+### Usage
+
+You can navigate through the navigation bar to see the different pages.
+
+### Deployment
+
+You can check the Github Pages link.
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@ichala](https://github.com/ichala/)
+- LinkedIn: [alijendoubi](https://www.linkedin.com/in/alijendoubi/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+-Thank you to Microverse Inc.
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] the link to the License is not working as expected, please have a look at [the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse for an example of how it is supposed to work.